static/src/stylesheets/module/experiments/_ab-acquisitions-love-boat.scss

```diff
@@ -0,0 +1,203 @@
+.contributions__epic {
+    &.contributions__epic--visual {
+        background: $neutral-8;
+    }
+
+    &.contributions__epic--visual--love {
+        border-top-color: $features-support-3;
+    }
+
+    &.contributions__epic--visual--boat {
+        border-top-color: $news-main-2;
+    }
+}
+
+.contributions__title.contributions__title--visual-epic {
+    @include fs-headline(7);
+    font-weight: bold;
+    margin-bottom: $gs-baseline * 3;
+}
+
+.contributions__title--visual-epic--boat {
+    color: $news-support-2;
+}
+
+.contributions__title--visual-epic {
+    @include fs-headline(7);
+    font-weight: bold;
+    margin-bottom: $gs-baseline * 3;
+}
+
+.contributions__title.contributions__title--visual-epic.contributions__title--visual-epic--step-2--love {
```

Comment:
I'll have to check on Monday. You are probably right, but I'm sure I came up against something like this where the modified class was being overridden by the first one, and I thought it was because they both had the same specificity (1 class). But this is my first time doing CSS for a while so I probably missed something 😆